### PR TITLE
Allow to change path when adding a new torrent

### PR DESCRIPTION
**Pull request by emersion**

### CODE REVIEW

File: src/ui/add_torrent.js [hunk 1]
```diff
@@ -180,14 +184,16 @@ class AddTorrent extends Component {
             id="new-torrent"
             start={start}
             startChanged={start => this.setState({ start })}
+            path={path}
+            pathChanged={path => this.setState({ path })}
```
Comment (SirCmpwn) [inline src/ui/add_torrent.js]: `setState({ path || null })`

File: src/ui/add_torrent.js [hunk 2]
```diff
@@ -180,14 +184,16 @@ class AddTorrent extends Component {
...
             priority={priority}
             priorityChanged={priority => this.setState({ priority })}
             downloadThrottle={downloadThrottle}
             downloadThrottleChanged={downloadThrottle =>
               this.setState({ downloadThrottle })}
             uploadThrottle={uploadThrottle}
             uploadThrottleChanged={uploadThrottle =>
-              { console.log(uploadThrottle); this.setState({ uploadThrottle }); }}
```
Comment (SirCmpwn) [inline src/ui/add_torrent.js]: derp

**Comment by emersion** [comment]
Done!

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**